docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
+| `Password`                          | no        | String  | null                      | The password for the session authentication<br>Mandatory if OpcAuthentication mode is UserName                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `DataSetWriterGroup`                | no        | String  | EndpointUrl               | The data set writer group collecting datasets defined for a certain <br>endpoint uniquely identified by the above attributes. <br>This attribute is used to identify the session opened into the <br>server. The default value consists of the EndpointUrl string, <br>followed by a deterministic hash composed of the <br>EndpointUrl, UseSecurity, OpcAuthenticationMode, UserName and Password attributes.                                                                                                                                                                                                          |
+| `DataSetWriterId`                   | no        | String  | DataSetPublishingInterval | The unique identifier for a data set writer used to collect <br>OPC UA nodes to be semantically grouped and published with <br>the same publishing interval. <br>When not specified a string representing the common <br>publishing interval of the nodes in the data set collection. <br>This attribute uniquely identifies a data set <br>within a DataSetWriterGroup. The uniqueness is determined <br>using the provided DataSetWriterId and the publishing <br>interval of the grouped OpcNodes.  An individual <br>subscription is created for each DataSetWriterId. |
+| `DataSetPublishingInterval`         | no        | Integer | null                      | The publishing interval used for a grouped set of nodes <br>under a certain DataSetWriter. When defined it <br>overrides the OpcPublishingInterval value in the OpcNodes <br>if grouped underneath a DataSetWriter. <br>_Note:_ the DataSetPublishingInterval is overwritten if an<br> OpcPublishingInterval is set at the node level                                                                                                                                                                                                                                  |
```

Comment:
><br> [](http://example.com/codeflow?start=146&length=4)

Why is <br> used? Markdown formatter should to the right thing? Only if a line break makes sense it might be used.

Reply:
I am assuming that you are referring to `<br>` as I cannot see the character in the comment. It helps a lot with formatting in tables and let's us avoid super long lines there. My markdown linter is also complaining about it locally, but GitHub renders it properly so we are making use of it.

Reply:
i understand, but it will unconditionally insert line breaks and will lead to not nicely formatted results when you have different with of screens.
I use Markdown All in One extension, which take care of the table formatting. I think it makes some sense when you want to start a new paragraph, like when showing Options or Default.
But at the end it is sort of style guide thing. On our spec i typically do not use it at all.